layers/+core/behavior/config.vim

```diff
@@ -0,0 +1,70 @@
+" call SpaceNeovimLoadFunc(expand('<sfile>:p'), 'func.vim')
+
+" Use system clipboard
+set clipboard=unnamed
+set noswapfile                      " Disable swap
+
+" Shorten the time before the vim-leader-guide buffer appears
+set timeoutlen=500
+set ttimeoutlen=0
+
+" UI {{{
+  set modeline                      " Read modelines from files
+  set laststatus=2
+  set list                          " Show invisible characters
+  set scrolloff=7                   " Set 7 lines to the cursor - when moving vertically using j/k
+  set splitright                    " Split on the right side
+  set splitbelow                    " Split below
+  set number                        " Show current line number
+  set relativenumber                " Enable relative line numbers
+" }}}
+
+" Whitespace, tabs & spaces {{{
+  set expandtab                     " Indent with spaces
+  set softtabstop=2                 " Number of spaces per <tab> when inserting
+  set shiftwidth=2                  " Number of spaces per <tab> when indenting
+  set tabstop=4                     " Number of spaces <tab> counts for
+" }}}
+
+" Backups {{{
+  set nobackup
```

Comment:
Hmm, I envision it more for crashes and then restoring from previous state, and not so much with undo. Does undo keep it persistent across file open and closes?